include/WCSimDetectorMessenger.hh

```diff
@@ -105,6 +105,9 @@ class WCSimDetectorMessenger: public G4UImessenger
   G4UIcmdWithAnInteger* PMTODperCellHorizontal;
   G4UIcmdWithAnInteger* PMTODperCellVertical;
   G4UIcmdWithADouble* PMTODPercentCoverage;
+  G4UIcmdWithADouble* PMTODPercentCoverageTop;
+  G4UIcmdWithADouble* PMTODPercentCoverageBottom;
+  G4UIcmdWithADouble* PMTODPercentCoverageBarrel;
```

Comment:
why not? what is changed in that version? is there no more TYVEK?

Reply:
sorry I was confused. Yes nothing to do with Tyvek.
Well the point of those options is that in the HK FD OD we are still trying to figure out how to optimally put the PMTs. You are right, in the "Realistic" version (by the way, a bad name in my opinion) the PMTs are assigned one by one, so one cannot separately define the coverages.

Reply:
Ahh sorry. I was under the impression that the PMT positions were finalised now. Since it effects e.g. how detector construction shifts are planned

I suppose what this means then is we need to keep two versions of the FD geometry
- `HyperK_HybridmPMT_WithOD_Realistic` which is used for most studies, and official MC productions
- `HyperK_HybridmPMT_WithOD` which is used only by the OD group for some dedicated studies
  - Do note that there are typically geometry overlaps/gaps with this geometry that can effect results

In both instances, the OD group should provide default options for their parameters


Thinking about it, yeah I agree "realistic" is premature. It is a massive improvement on the old geometry (recall [Menai's mPMT plot fig 1 here](https://wiki.hyperk.org/pub/HyperK/TechNote0075/Tech_Note_HK_Data_Rate.pdf)), but with a long-term view it "based_on_Autumn_2023_integration_diagrams" is more accurate.

Anyway... it's the last _planned_ HKFD geometry in WCSim. So I think we can live with this name

Reply:
Yes I agree with all you say. You may be right that we won't change the PMT positions anymore, although they still ask me to check other options every now and then.
Yes I am aware of the geometry overlap and I am very happy to have a version without overlaps. I think the main point of this PR is to reach an OD simulation that has both the right PMT positions (Patrick work) and the right parameters (my work)

Reply:
That sounds great! 👍 
Closing this thread now